daemon/src/projection.rs

```diff
@@ -567,7 +564,7 @@ impl Cfd {
         self
     }
 
-    pub fn with_current_quote(self, latest_quote: Option<xtra_bitmex_price_feed::Quote>) -> Self {
+    pub fn with_current_quote(self, latest_quotes: Option<&LatestQuotes>) -> Self {
```

Comment:
🔧 We can remove the `Option`, right? And empty `HashMap` will have the same meaning as `None`.

Reply:
Technically yes, I opted against it because an empty `HashMap` has a semantically different meaning in this context IMHO:
1) we don't want to use quotes (passing `None`)
2) there were no quotes - we wanted to use quotes, but there were none.

Reply:
Yep, you are right! We do actually use case 1.